### DISCUSSION TITLE
Cannot set 5.6K/6K 360 30fps for MAX

### DISCUSSION THREAD

**Discussion by attila0307** [gopro/labs#669] [General]
As the description says, only 3K 360 can be set from the Labs QR.
Can you add the 5.6K/6K stitched option to the list?

**Comment by dnewman-gpsw** [comment]
Description doesn't exactly say that.  5K selection work on all cameras that have a 5.x resolution. 5.6K for MAX is the same command and 5.3K for HERO9-12.  Command is r5.

**Comment by attila0307** [comment]
Great, thank you very much!
